src/main/java/crazypants/enderio/machine/transceiver/TileTransceiver.java

```diff
@@ -48,6 +48,7 @@
   private final SetMultimap<ChannelType, Channel> sendChannels = MultimapBuilder.enumKeys(ChannelType.class).hashSetValues().build();
   private final SetMultimap<ChannelType, Channel> recieveChannels = MultimapBuilder.enumKeys(ChannelType.class).hashSetValues().build();
 
+  private final ICapacitor capacitor = new BasicCapacitor(0, Config.transceiverMaxIoRF * 2, 500000, Config.transceiverMaxIoRF);
```

Comment:
maybe this should be a static, to better indicate that this is a constant, wdyt?

Reply:
This is actually not constant because `BasicCapacitor` is a mutable class, so we probably do want to create a new one for each instance of `TileTransceiver`